awesome.md

```diff
@@ -0,0 +1,54 @@
+# The awesome manifesto
+
+If you want your list to be included on `awesome`, you should really 
```

Comment:
> If you want your list to be included on `awesome`, you should really  just include awesome stuff.

This makes me think this is a list of rules to be included in the list. It should just be a list of tips/guidelines about how to make your lists more awesome.